UnitTests/View/Layout/Dim.AutoTests.cs

```diff
@@ -203,40 +204,35 @@ public void SubView_Changes_SuperView_Size (int subX, int subY, int subWidth, in
     [Fact]
     public void TestEquality ()
     {
-        var a = new DimAuto
-        {
-            MaximumContentDim = null,
-            MinimumContentDim = 1,
-            Style = DimAutoStyle.Auto
-        };
-
-        var b = new DimAuto
-        {
-            MaximumContentDim = null,
-            MinimumContentDim = 1,
-            Style = DimAutoStyle.Auto
-        };
-
-        var c = new DimAuto
-        {
-            MaximumContentDim = 2,
-            MinimumContentDim = 1,
-            Style = DimAutoStyle.Auto
-        };
-
-        var d = new DimAuto
-        {
-            MaximumContentDim = null,
-            MinimumContentDim = 1,
-            Style = DimAutoStyle.Content
-        };
-
-        var e = new DimAuto
-        {
-            MaximumContentDim = null,
-            MinimumContentDim = 2,
-            Style = DimAutoStyle.Auto
-        };
+        var a = new DimAuto (
```

Comment:
I only updated this test to show that you can call them with named parameters, too, if you want, which looks almost exactly like an initializer.

Otherwise, you can just call them like normal positional parameters.

This whole test method is irrelevant now that these are records anyway, though. Just showing off records for your benefit. 🙂